src/modules/python/pbs/v1/_svr_types.py

```diff
@@ -94,151 +96,155 @@ def get_local_nodename():
 #                  the job must be in the queue 'filter_queue' for the
 #                  job object to be instantiated.
 def pbs_statobj(type, name=None, connect_server=None, filter_queue=None):
```

Comment:
if possible then rename `type` to `objtype` as `type` is a builtin method in Python